### ISSUE TITLE
(Feature Request) Option to use the old app menu from version 21.04.

### ISSUE THREAD

**Issue by Mesum-Hussain** [pop-os/gnome-shell-extension-pop-cosmic#281]
I don't really like the design of the new application menu and it decreases my mouse efficiency because of smaller target icons.
Also the old app menu integrated well with the "Blur My Shell" extension to make it very beautiful.

The new app menu is also lacking the functionality of draging mouse from the left side of the screen to launch it, which decreases my efficiency when I am using an external mouse (not a laptop trackpad). And I think it kind of defeats the purpose of an app menu because app menu as I see it should work without me needing to lift my hand from mouse, if I need to press a keyboard shortcut to bring it then I would much rather use the search launcher (which I have been doing a lot since 21.10. released). And I think adding the application button in the top bar just incaseas the clutter in the top bar; Maybe a double right click would be a better solution, but that's a different discussion.
Speaking design wise, it looks very ugly compared to the old one. And if you had your apps well organised and everything is groped relevantly (like in my case), then you will have two trails of file icons running across the bottom of the app menu window, stacked on top of each other, which doesn't look minimalistic at all and look rather ugly. So in this new menu design there is a trade off between organisation and minimalistic look, which wasn't the case with the old app menu (in fact, the more you organised it, the more it used to look minimalistic). And it doesn't integrate well with the "Blur My Shell" extension which makes my system look inconsistent as everything is having a Blur-glass morden look except for the app menu which looks very archaic.

Basically I like all the other features of the Cosmic desktop except for the new application menu. And used to love the old app menu from 21.04. I want to request a toggle button in the settings which lets users decide which of the two app menus they would like to use.

**Comment by synergic777** [comment]
Totally agree!
-  Also folder icons are useless right now. The previous app menu was displaying 4 small miniatures of apps inside the folder.  Now, this repeatedly used icon of the folder doesn't help at all in navigating. 
-  Furthermore, the "Available to install" section in the search doesn't disappear when there aren't any results.
- Remove/edit name buttons have a weird nonproportional shape, but to be honest I don't think that we need the first one at all. Let's use the rule "do more with less" - double click on a name to edit it.
- I think that there is needed a border that would divide the app library from the rest of the content on the screen as well. Blur would be great, but System76 apparently doesn't like this solution. Without it, on the screen happens to many things, that's not clear at all. Maybe a dark vignette similar to that which pops out during delete action?
- The search bar has "Type to search" text inside it, but typing cursor starts a few characters before - that doesn't look good.
I appreciate the idea, but it still needs a lot of work.

Overall, I feel the previous app library was a more intuitive, more thoughtful, and good-looking app library. It wasn't perfect, but it was doing a lot of things correctly. I hope that Pop!_OS's library gonna evolve as well.  

 "When you really love something, You don't go easy on it, you actually critique it more". 
Regards, Martin
